src/corona_hakab_model/supervisor.py

```diff
@@ -47,13 +47,10 @@ def snapshot(self, manager):
             s.snapshot(manager)
 
     def dump(self, filename=None):
-        file_name = Path(filename) if filename else Path(SIM_OUTPUT_FOLDER) / (datetime.now().strftime("%Y%m%d-%H%M%S") + ".csv")
+        output_folder = Path(SIM_OUTPUT_FOLDER)
```

Comment:
Why do you use Path here? SIM_OUTPUT_FOLDER is supposed to be a path not str

Reply:
SIM_OUTPUT_FOLDER is already a path object, so you can use it as is

Reply:
Okay, changed it in the latest commit